app/src/ui/cherry-pick/cherry-pick-conflicts-dialog.tsx

```diff
@@ -128,7 +128,7 @@ export class CherryPickConflictsDialog extends React.Component<
 
     const {
       manualResolutions,
-      targetBranchName: theirBranch,
+      targetBranchName: ourBranch,
```

Comment:
Is this change correct in all cases? I ask because my brain always 🤯  when I try to think about their vs our branch in git conflicts 😂

Reply:
:D - Yes. I had them backwards 🤦  from the start because rebase requires them opposite and I was following it as a pattern and was a little confused by them so I thought I just didn't understand them at the time.

Reply:
Btw, AFAIK, the only other cherry-picking scenario is a regular conflict and I did test that the wording made sense for the outcome that followed. :)